spec/features/admin/users_spec.rb

```diff
@@ -17,7 +17,7 @@
     before do
       create(:user)
       create(:office, head: user)
-      create(:allocation, user: allocated_user)
+      create(:allocation, user: allocated_user, ongoing: true)
```

Comment:
maybe we should set the `ongoing` value for allocation as true on the factory, feels kinda counter-intuitive to have the field with the default value `true` and its factory as `false` https://github.com/Codeminer42/Punchclock/blob/9a554fc2b428ec6bd0a524dbcc0baa4f86a30883/spec/factories/allocations.rb#L10